staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3988,13 +3988,34 @@ type LoadBalancerIngress struct {
 	// +listType=atomic
 	// +optional
 	Ports []PortStatus `json:"ports,omitempty" protobuf:"bytes,4,rep,name=ports"`
+
+	// IPMode specifies how the load-balancer's IP behaves.
+	// Setting this to "VIP" indicates that the traffic passing through
+	// this load-balancer is delivered with the destination IP and port set to the load-balancer's IP and port.
+	// Setting this to "Proxy" indicates that the load-balancer acts like a proxy,
+	// delivering traffic with the destination IP and port set to the node's IP and nodePort or to the pod's IP and targetPort.
+	// This field can only be set when the ip field is also set, and defaults to "VIP" if not specified.
+	// +optional
+	IPMode *LoadBalancerIPMode `json:"ipMode,omitempty" protobuf:"bytes,3,opt,name=ipMode"`
```

Comment:
I am concerned that we are using tag 3 here, when tag 4 is already used.  I can't see why PortStatus would have not used 3, and I must have missed that at review time.

@janosi - you added PortStatus in c970a46bc1bc - did you use 4 for a reason?

Reply:
Interesting, I remember that both PR were close in time, and this had to be reverted  https://github.com/kubernetes/kubernetes/pull/96454 , is it possible that is the reason? that this PR predated the PortStatus one?

Reply:
So should I bump to 5 ? or keep 3? 🤔

Reply:
Ahh, I am sorry :( I am pretty sure I missed to fallback to 3 when the other PR was reverted :(

Reply:
It's fine, as long as we understand what happened :)